prisma/schema.prisma

```diff
@@ -4,7 +4,7 @@ generator client {
 
 datasource db {
   provider = "postgresql"
-  url = env("DATABASE_URL")
+  url = "postgres://test:test@localhost:5432/test"
```

Comment:
(╯°□°)╯︵ ┻━┻ 💥

Reply:
There is a bug that we still don't understand, but there is a local error that doesn't accept this reference. However, on the remote side, it is accepted. I will push it corrected, and then I will investigate again to find out what might be causing it. Another sticker unlocked!